doc/en/components/grids/_shared/export-excel.md

```diff
@@ -0,0 +1,234 @@
+---
+title: {Platform} {Component} Export to Excel - {ProductName} for {Platform}
+_description: With {ProductName} Excel Exporter you can make client Excel functionality more convenient & simpler. This format allows features like filtering, sorting, etc.
+_keywords: {Platform}, {ComponentTitle}, {ComponentTitle}, {ProductName}, Infragistics
+mentionedTypes: [{ComponentApiMembers}]
+sharedComponents: ["Grid", "TreeGrid", "HierarchicalGrid"]
+---
+
+# {Platform} {Component} Export to Excel Service
+
+
+<p class="highlight">
+  The Excel Exporter service can export data to excel from the {Component}. The data export functionality is encapsulated in the [`ExcelExporterService`]({environment:demosBaseUrl}/classes/excelexporterservice.html) class and the data is exported in MS Excel table format. This format allows features like filtering, sorting, etc. To do this you need to invoke the [`ExcelExporterService`]({environment:demosBaseUrl}/classes/excelexporterservice.html)'s [`export`]({environment:demosBaseUrl}/classes/excelexporterservice.html#export) method and pass the {ComponentTitle} component as first argument to export grid easily.
+</p>
```

Comment:
remove all hard-code links and just use API members (in camelcase) and wrap them in backticks, e.g. ``` `ExcelExporterService` ``` ``` `Export` ```